CRITICAL - 2023-02-02 01:28:34 --> The framework needs the following extension(s) installed and loaded: intl, mbstring.
in SYSTEMPATH\CodeIgniter.php on line 226.
 1 SYSTEMPATH\CodeIgniter.php(226): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl, mbstring')
 2 SYSTEMPATH\CodeIgniter.php(190): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 ROOTPATH\spark(83): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2023-02-02 01:28:34 --> Uncaught Error: Call to undefined function CodeIgniter\CLI\mb_strpos() in E:\Codigo\TareaAPI\system\CLI\CLI.php:872
Stack trace:
#0 E:\Codigo\TareaAPI\system\CLI\CLI.php(161): CodeIgniter\CLI\CLI::parseCommandLine()
#1 E:\Codigo\TareaAPI\system\CLI\CLI.php(1115): CodeIgniter\CLI\CLI::init()
#2 E:\Codigo\TareaAPI\system\Autoloader\Autoloader.php(312): include_once('E:\\Codigo\\Tarea...')
#3 E:\Codigo\TareaAPI\system\Autoloader\Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile('E:\\Codigo\\Tarea...')
#4 E:\Codigo\TareaAPI\system\Autoloader\Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace('CodeIgniter\\CLI...')
#5 E:\Codigo\TareaAPI\app\Views\errors\cli\error_exception.php(6): CodeIgniter\Autoloader\Autoloader->loadClass('CodeIgniter\\CLI...')
#6 E:\Codigo\TareaAPI\app\Views\errors\cli\production.php(5): include('E:\\Codigo\\Tarea...')
#7 E:\Codigo\TareaAPI\system\Debug\Exceptions.php(266): include('E:\\Codigo\\Tarea...')
#8 E:\Codigo\TareaAPI\system\Debug\Exceptions.php(269): CodeIgniter\Debug\Exceptions->CodeIgniter\Debug\{closure}()
#9 E:\Codigo\TareaAPI\system\Debug\Exceptions.php(147): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Exceptions\FrameworkException), 500)
#10 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\FrameworkException))
#11 {main}
  thrown
in SYSTEMPATH\CLI\CLI.php on line 872.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2023-02-02 03:45:40 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:45:48 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:46:00 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:46:01 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:46:02 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:46:32 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:46:36 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:47:06 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:48:21 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:48:29 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:48:42 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:53:13 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:53:25 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:53:27 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:53:28 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:53:29 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:57:09 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:57:11 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:57:11 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:57:24 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:57:25 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:57:37 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:57:38 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:57:52 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:57:53 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:57:54 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 03:57:55 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:00:07 --> Class "App\Models\IndicadorFinancieroModel" not found
in APPPATH\Controllers\Home.php on line 11.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:02:15 --> Undefined variable $db
in APPPATH\Controllers\Home.php on line 11.
 1 APPPATH\Controllers\Home.php(11): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $db', 'APPPATH\\Controllers\\Home.php', 11)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:03:57 --> Undefined variable $db
in APPPATH\Controllers\Home.php on line 11.
 1 APPPATH\Controllers\Home.php(11): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $db', 'APPPATH\\Controllers\\Home.php', 11)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:04:03 --> Undefined variable $db
in APPPATH\Controllers\Home.php on line 11.
 1 APPPATH\Controllers\Home.php(11): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $db', 'APPPATH\\Controllers\\Home.php', 11)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:04:16 --> Undefined variable $db
in APPPATH\Controllers\Home.php on line 11.
 1 APPPATH\Controllers\Home.php(11): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $db', 'APPPATH\\Controllers\\Home.php', 11)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:10:39 --> Class "CodeIgniter\Database\Postgres\Connection" not found
in SYSTEMPATH\Database\Database.php on line 138.
 1 SYSTEMPATH\Database\Database.php(56): CodeIgniter\Database\Database->initDriver('Postgres', 'CodeIgniter\\Database\\Postgres\\Connection', [...])
 2 SYSTEMPATH\Database\Config.php(82): CodeIgniter\Database\Database->load([...], 'default')
 3 SYSTEMPATH\Common.php(352): CodeIgniter\Database\Config::connect('default', true)
 4 APPPATH\Controllers\BaseController.php(55): db_connect()
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:11:16 --> Undefined variable $db
in APPPATH\Controllers\Home.php on line 11.
 1 APPPATH\Controllers\Home.php(11): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $db', 'APPPATH\\Controllers\\Home.php', 11)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:11:22 --> Undefined variable $db
in APPPATH\Controllers\Home.php on line 11.
 1 APPPATH\Controllers\Home.php(11): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $db', 'APPPATH\\Controllers\\Home.php', 11)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:11:24 --> Undefined variable $db
in APPPATH\Controllers\Home.php on line 11.
 1 APPPATH\Controllers\Home.php(11): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $db', 'APPPATH\\Controllers\\Home.php', 11)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:11:31 --> Class "CodeIgniter\Database\Postgres\Connection" not found
in SYSTEMPATH\Database\Database.php on line 138.
 1 SYSTEMPATH\Database\Database.php(56): CodeIgniter\Database\Database->initDriver('Postgres', 'CodeIgniter\\Database\\Postgres\\Connection', [...])
 2 SYSTEMPATH\Database\Config.php(82): CodeIgniter\Database\Database->load([...], 'default')
 3 SYSTEMPATH\Common.php(352): CodeIgniter\Database\Config::connect('default', true)
 4 APPPATH\Controllers\BaseController.php(55): db_connect()
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:13:15 --> Class "CodeIgniter\Database\Postgres\Connection" not found
in SYSTEMPATH\Database\Database.php on line 138.
 1 SYSTEMPATH\Database\Database.php(56): CodeIgniter\Database\Database->initDriver('Postgres', 'CodeIgniter\\Database\\Postgres\\Connection', [...])
 2 SYSTEMPATH\Database\Config.php(82): CodeIgniter\Database\Database->load([...], 'default')
 3 SYSTEMPATH\Common.php(352): CodeIgniter\Database\Config::connect('default', true)
 4 APPPATH\Controllers\Home.php(10): db_connect()
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:13:17 --> Class "CodeIgniter\Database\Postgres\Connection" not found
in SYSTEMPATH\Database\Database.php on line 138.
 1 SYSTEMPATH\Database\Database.php(56): CodeIgniter\Database\Database->initDriver('Postgres', 'CodeIgniter\\Database\\Postgres\\Connection', [...])
 2 SYSTEMPATH\Database\Config.php(82): CodeIgniter\Database\Database->load([...], 'default')
 3 SYSTEMPATH\Common.php(352): CodeIgniter\Database\Config::connect('default', true)
 4 APPPATH\Controllers\Home.php(10): db_connect()
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:13:22 --> Class "CodeIgniter\Database\Postgres\Connection" not found
in SYSTEMPATH\Database\Database.php on line 138.
 1 SYSTEMPATH\Database\Database.php(56): CodeIgniter\Database\Database->initDriver('Postgres', 'CodeIgniter\\Database\\Postgres\\Connection', [...])
 2 SYSTEMPATH\Database\Config.php(82): CodeIgniter\Database\Database->load([...], 'default')
 3 SYSTEMPATH\Common.php(352): CodeIgniter\Database\Config::connect('default', true)
 4 APPPATH\Controllers\Home.php(10): db_connect()
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:15:13 --> You must set the database table to be used with your query.
in SYSTEMPATH\Database\BaseConnection.php on line 882.
 1 SYSTEMPATH\Model.php(641): CodeIgniter\Database\BaseConnection->table('ROOTPATH\\public')
 2 SYSTEMPATH\Model.php(187): CodeIgniter\Model->builder()
 3 SYSTEMPATH\BaseModel.php(557): CodeIgniter\Model->doFind(true, 1)
 4 APPPATH\Controllers\Home.php(13): CodeIgniter\BaseModel->find(1)
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:15:15 --> You must set the database table to be used with your query.
in SYSTEMPATH\Database\BaseConnection.php on line 882.
 1 SYSTEMPATH\Model.php(641): CodeIgniter\Database\BaseConnection->table('ROOTPATH\\public')
 2 SYSTEMPATH\Model.php(187): CodeIgniter\Model->builder()
 3 SYSTEMPATH\BaseModel.php(557): CodeIgniter\Model->doFind(true, 1)
 4 APPPATH\Controllers\Home.php(13): CodeIgniter\BaseModel->find(1)
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:17:21 --> You must set the database table to be used with your query.
in SYSTEMPATH\Database\BaseConnection.php on line 882.
 1 SYSTEMPATH\Model.php(641): CodeIgniter\Database\BaseConnection->table('ROOTPATH\\public')
 2 SYSTEMPATH\Model.php(187): CodeIgniter\Model->builder()
 3 SYSTEMPATH\BaseModel.php(557): CodeIgniter\Model->doFind(true, '1')
 4 APPPATH\Controllers\Home.php(13): CodeIgniter\BaseModel->find('1')
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:17:28 --> You must set the database table to be used with your query.
in SYSTEMPATH\Database\BaseConnection.php on line 882.
 1 SYSTEMPATH\Model.php(641): CodeIgniter\Database\BaseConnection->table('ROOTPATH\\public')
 2 SYSTEMPATH\Model.php(187): CodeIgniter\Model->builder()
 3 SYSTEMPATH\BaseModel.php(557): CodeIgniter\Model->doFind(true, '1')
 4 APPPATH\Controllers\Home.php(13): CodeIgniter\BaseModel->find('1')
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:17:29 --> You must set the database table to be used with your query.
in SYSTEMPATH\Database\BaseConnection.php on line 882.
 1 SYSTEMPATH\Model.php(641): CodeIgniter\Database\BaseConnection->table('ROOTPATH\\public')
 2 SYSTEMPATH\Model.php(187): CodeIgniter\Model->builder()
 3 SYSTEMPATH\BaseModel.php(557): CodeIgniter\Model->doFind(true, '1')
 4 APPPATH\Controllers\Home.php(13): CodeIgniter\BaseModel->find('1')
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:17:30 --> You must set the database table to be used with your query.
in SYSTEMPATH\Database\BaseConnection.php on line 882.
 1 SYSTEMPATH\Model.php(641): CodeIgniter\Database\BaseConnection->table('ROOTPATH\\public')
 2 SYSTEMPATH\Model.php(187): CodeIgniter\Model->builder()
 3 SYSTEMPATH\BaseModel.php(557): CodeIgniter\Model->doFind(true, '1')
 4 APPPATH\Controllers\Home.php(13): CodeIgniter\BaseModel->find('1')
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:19:10 --> You must set the database table to be used with your query.
in SYSTEMPATH\Database\BaseConnection.php on line 882.
 1 SYSTEMPATH\Model.php(641): CodeIgniter\Database\BaseConnection->table('ROOTPATH\\public')
 2 SYSTEMPATH\Model.php(187): CodeIgniter\Model->builder()
 3 SYSTEMPATH\BaseModel.php(557): CodeIgniter\Model->doFind(true, '1')
 4 APPPATH\Controllers\Home.php(13): CodeIgniter\BaseModel->find('1')
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:22:01 --> You must set the database table to be used with your query.
in SYSTEMPATH\Database\BaseConnection.php on line 882.
 1 SYSTEMPATH\Model.php(641): CodeIgniter\Database\BaseConnection->table('ROOTPATH\\public')
 2 SYSTEMPATH\Model.php(187): CodeIgniter\Model->builder()
 3 SYSTEMPATH\BaseModel.php(557): CodeIgniter\Model->doFind(true, '1')
 4 APPPATH\Controllers\Home.php(13): CodeIgniter\BaseModel->find('1')
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:23:23 --> You must set the database table to be used with your query.
in SYSTEMPATH\Database\BaseConnection.php on line 882.
 1 SYSTEMPATH\Model.php(641): CodeIgniter\Database\BaseConnection->table('ROOTPATH\\public')
 2 SYSTEMPATH\Model.php(187): CodeIgniter\Model->builder()
 3 SYSTEMPATH\BaseModel.php(557): CodeIgniter\Model->doFind(true, '1')
 4 APPPATH\Controllers\Home.php(13): CodeIgniter\BaseModel->find('1')
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:23:36 --> You must set the database table to be used with your query.
in SYSTEMPATH\Database\BaseConnection.php on line 882.
 1 SYSTEMPATH\Model.php(641): CodeIgniter\Database\BaseConnection->table('ROOTPATH\\public')
 2 SYSTEMPATH\Model.php(187): CodeIgniter\Model->builder()
 3 SYSTEMPATH\BaseModel.php(557): CodeIgniter\Model->doFind(true, '1')
 4 APPPATH\Controllers\Home.php(13): CodeIgniter\BaseModel->find('1')
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:23:39 --> You must set the database table to be used with your query.
in SYSTEMPATH\Database\BaseConnection.php on line 882.
 1 SYSTEMPATH\Model.php(641): CodeIgniter\Database\BaseConnection->table('ROOTPATH\\public')
 2 SYSTEMPATH\Model.php(187): CodeIgniter\Model->builder()
 3 SYSTEMPATH\BaseModel.php(557): CodeIgniter\Model->doFind(true, '1')
 4 APPPATH\Controllers\Home.php(13): CodeIgniter\BaseModel->find('1')
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-02-02 04:25:38 --> Error connecting to the database: ErrorException: pg_connect(): Unable to connect to PostgreSQL server: fe_sendauth: no password supplied in E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php:75
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_connect(): U...', 'E:\\Codigo\\Tarea...', 75)
#1 E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php(75): pg_connect('host=localhost ...')
#2 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(391): CodeIgniter\Database\Postgre\Connection->connect(false)
#3 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#4 E:\Codigo\TareaAPI\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 E:\Codigo\TareaAPI\system\Model.php(199): CodeIgniter\Database\BaseBuilder->get()
#6 E:\Codigo\TareaAPI\system\BaseModel.php(557): CodeIgniter\Model->doFind(true, '1')
#7 E:\Codigo\TareaAPI\app\Controllers\Home.php(13): CodeIgniter\BaseModel->find('1')
#8 E:\Codigo\TareaAPI\system\CodeIgniter.php(934): App\Controllers\Home->index()
#9 E:\Codigo\TareaAPI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 E:\Codigo\TareaAPI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\Codigo\TareaAPI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 E:\Codigo\TareaAPI\system\Commands\Server\rewrite.php(47): require_once('E:\\Codigo\\Tarea...')
#13 {main}
CRITICAL - 2023-02-02 04:25:38 --> Unable to connect to the database.
Main connection [Postgre]: pg_connect(): Unable to connect to PostgreSQL server: fe_sendauth: no password supplied
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM "indicadorfinancieros"
WHERE "indicadorfinancieros"."id" = :indicadorfinancieros.id:', [...], false)
 3 SYSTEMPATH\Model.php(199): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(557): CodeIgniter\Model->doFind(true, '1')
 5 APPPATH\Controllers\Home.php(13): CodeIgniter\BaseModel->find('1')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-02-02 04:26:07 --> Error connecting to the database: ErrorException: pg_connect(): Unable to connect to PostgreSQL server: fe_sendauth: no password supplied in E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php:75
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_connect(): U...', 'E:\\Codigo\\Tarea...', 75)
#1 E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php(75): pg_connect('host=localhost ...')
#2 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(391): CodeIgniter\Database\Postgre\Connection->connect(false)
#3 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#4 E:\Codigo\TareaAPI\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 E:\Codigo\TareaAPI\system\Model.php(199): CodeIgniter\Database\BaseBuilder->get()
#6 E:\Codigo\TareaAPI\system\BaseModel.php(557): CodeIgniter\Model->doFind(true, '1')
#7 E:\Codigo\TareaAPI\app\Controllers\Home.php(13): CodeIgniter\BaseModel->find('1')
#8 E:\Codigo\TareaAPI\system\CodeIgniter.php(934): App\Controllers\Home->index()
#9 E:\Codigo\TareaAPI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 E:\Codigo\TareaAPI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\Codigo\TareaAPI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 E:\Codigo\TareaAPI\system\Commands\Server\rewrite.php(47): require_once('E:\\Codigo\\Tarea...')
#13 {main}
CRITICAL - 2023-02-02 04:26:07 --> Unable to connect to the database.
Main connection [Postgre]: pg_connect(): Unable to connect to PostgreSQL server: fe_sendauth: no password supplied
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM "indicadorfinancieros"
WHERE "indicadorfinancieros"."id" = :indicadorfinancieros.id:', [...], false)
 3 SYSTEMPATH\Model.php(199): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(557): CodeIgniter\Model->doFind(true, '1')
 5 APPPATH\Controllers\Home.php(13): CodeIgniter\BaseModel->find('1')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-02-02 04:26:12 --> Error connecting to the database: ErrorException: pg_connect(): Unable to connect to PostgreSQL server: fe_sendauth: no password supplied in E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php:75
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_connect(): U...', 'E:\\Codigo\\Tarea...', 75)
#1 E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php(75): pg_connect('host=localhost ...')
#2 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(391): CodeIgniter\Database\Postgre\Connection->connect(false)
#3 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#4 E:\Codigo\TareaAPI\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 E:\Codigo\TareaAPI\system\Model.php(199): CodeIgniter\Database\BaseBuilder->get()
#6 E:\Codigo\TareaAPI\system\BaseModel.php(557): CodeIgniter\Model->doFind(true, '1')
#7 E:\Codigo\TareaAPI\app\Controllers\Home.php(13): CodeIgniter\BaseModel->find('1')
#8 E:\Codigo\TareaAPI\system\CodeIgniter.php(934): App\Controllers\Home->index()
#9 E:\Codigo\TareaAPI\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 E:\Codigo\TareaAPI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\Codigo\TareaAPI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 E:\Codigo\TareaAPI\system\Commands\Server\rewrite.php(47): require_once('E:\\Codigo\\Tarea...')
#13 {main}
CRITICAL - 2023-02-02 04:26:12 --> Unable to connect to the database.
Main connection [Postgre]: pg_connect(): Unable to connect to PostgreSQL server: fe_sendauth: no password supplied
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM "indicadorfinancieros"
WHERE "indicadorfinancieros"."id" = :indicadorfinancieros.id:', [...], false)
 3 SYSTEMPATH\Model.php(199): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(557): CodeIgniter\Model->doFind(true, '1')
 5 APPPATH\Controllers\Home.php(13): CodeIgniter\BaseModel->find('1')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:26:20 --> Undefined variable $var_dump
in APPPATH\Controllers\Home.php on line 14.
 1 APPPATH\Controllers\Home.php(14): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $var_dump', 'APPPATH\\Controllers\\Home.php', 14)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:26:33 --> Undefined variable $var_dump
in APPPATH\Controllers\Home.php on line 14.
 1 APPPATH\Controllers\Home.php(14): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $var_dump', 'APPPATH\\Controllers\\Home.php', 14)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 04:26:33 --> Undefined variable $var_dump
in APPPATH\Controllers\Home.php on line 14.
 1 APPPATH\Controllers\Home.php(14): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $var_dump', 'APPPATH\\Controllers\\Home.php', 14)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 16:43:41 --> Undefined variable $db
in APPPATH\Controllers\Home.php on line 12.
 1 APPPATH\Controllers\Home.php(12): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $db', 'APPPATH\\Controllers\\Home.php', 12)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 16:43:42 --> Undefined variable $db
in APPPATH\Controllers\Home.php on line 12.
 1 APPPATH\Controllers\Home.php(12): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $db', 'APPPATH\\Controllers\\Home.php', 12)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 16:43:43 --> Undefined variable $db
in APPPATH\Controllers\Home.php on line 12.
 1 APPPATH\Controllers\Home.php(12): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $db', 'APPPATH\\Controllers\\Home.php', 12)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 16:43:44 --> Undefined variable $db
in APPPATH\Controllers\Home.php on line 12.
 1 APPPATH\Controllers\Home.php(12): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $db', 'APPPATH\\Controllers\\Home.php', 12)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 16:43:47 --> Undefined variable $db
in APPPATH\Controllers\Home.php on line 12.
 1 APPPATH\Controllers\Home.php(12): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $db', 'APPPATH\\Controllers\\Home.php', 12)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 16:43:49 --> Undefined variable $db
in APPPATH\Controllers\Home.php on line 12.
 1 APPPATH\Controllers\Home.php(12): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $db', 'APPPATH\\Controllers\\Home.php', 12)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 16:45:41 --> Undefined variable $db
in APPPATH\Controllers\Home.php on line 12.
 1 APPPATH\Controllers\Home.php(12): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $db', 'APPPATH\\Controllers\\Home.php', 12)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 16:45:45 --> Undefined variable $db
in APPPATH\Controllers\Home.php on line 12.
 1 APPPATH\Controllers\Home.php(12): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $db', 'APPPATH\\Controllers\\Home.php', 12)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 16:45:58 --> Undefined property: App\Controllers\Home::$db
in APPPATH\Controllers\Home.php on line 12.
 1 APPPATH\Controllers\Home.php(12): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Controllers\\Home::$db', 'APPPATH\\Controllers\\Home.php', 12)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 18:49:24 --> CURL must be enabled to use the CURLRequest class.
in SYSTEMPATH\HTTP\CURLRequest.php on line 103.
 1 SYSTEMPATH\HTTP\CURLRequest.php(103): CodeIgniter\HTTP\Exceptions\HTTPException::forMissingCurl()
 2 SYSTEMPATH\Config\Services.php(198): CodeIgniter\HTTP\CURLRequest->__construct(Object(Config\App), Object(CodeIgniter\HTTP\URI), Object(CodeIgniter\HTTP\Response), [])
 3 SYSTEMPATH\Config\BaseService.php(252): CodeIgniter\Config\Services::curlrequest([], Object(CodeIgniter\HTTP\Response), Object(Config\App), false)
 4 SYSTEMPATH\Config\BaseService.php(193): CodeIgniter\Config\BaseService::__callStatic('curlrequest', [...])
 5 SYSTEMPATH\Config\Services.php(192): CodeIgniter\Config\BaseService::getSharedInstance('curlrequest', [], null, null)
 6 SYSTEMPATH\Config\BaseService.php(252): CodeIgniter\Config\Services::curlrequest()
 7 APPPATH\Controllers\BaseController.php(75): CodeIgniter\Config\BaseService::__callStatic('curlrequest', [])
 8 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 9 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 18:50:34 --> 60 : SSL certificate problem: unable to get local issuer certificate
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('60', 'SSL certificate problem: unable to get local issuer certificate')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 18:50:36 --> 60 : SSL certificate problem: unable to get local issuer certificate
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('60', 'SSL certificate problem: unable to get local issuer certificate')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 18:50:39 --> 60 : SSL certificate problem: unable to get local issuer certificate
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('60', 'SSL certificate problem: unable to get local issuer certificate')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 18:50:41 --> 60 : SSL certificate problem: unable to get local issuer certificate
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('60', 'SSL certificate problem: unable to get local issuer certificate')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 18:50:42 --> 60 : SSL certificate problem: unable to get local issuer certificate
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('60', 'SSL certificate problem: unable to get local issuer certificate')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:00:39 --> Undefined array key "ssl_key"
in SYSTEMPATH\HTTP\CURLRequest.php on line 531.
 1 SYSTEMPATH\HTTP\CURLRequest.php(531): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "ssl_key"', 'SYSTEMPATH\\HTTP\\CURLRequest.php', 531)
 2 SYSTEMPATH\HTTP\CURLRequest.php(363): CodeIgniter\HTTP\CURLRequest->setCURLOptions([...], [...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:04:39 --> Undefined array key "ssl_key"
in SYSTEMPATH\HTTP\CURLRequest.php on line 531.
 1 SYSTEMPATH\HTTP\CURLRequest.php(531): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "ssl_key"', 'SYSTEMPATH\\HTTP\\CURLRequest.php', 531)
 2 SYSTEMPATH\HTTP\CURLRequest.php(363): CodeIgniter\HTTP\CURLRequest->setCURLOptions([...], [...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:04:55 --> Undefined array key "ssl_key"
in SYSTEMPATH\HTTP\CURLRequest.php on line 531.
 1 SYSTEMPATH\HTTP\CURLRequest.php(531): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "ssl_key"', 'SYSTEMPATH\\HTTP\\CURLRequest.php', 531)
 2 SYSTEMPATH\HTTP\CURLRequest.php(363): CodeIgniter\HTTP\CURLRequest->setCURLOptions([...], [...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:04:59 --> 60 : SSL certificate problem: unable to get local issuer certificate
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('60', 'SSL certificate problem: unable to get local issuer certificate')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:05:13 --> Undefined array key "ssl_key"
in SYSTEMPATH\HTTP\CURLRequest.php on line 531.
 1 SYSTEMPATH\HTTP\CURLRequest.php(531): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "ssl_key"', 'SYSTEMPATH\\HTTP\\CURLRequest.php', 531)
 2 SYSTEMPATH\HTTP\CURLRequest.php(363): CodeIgniter\HTTP\CURLRequest->setCURLOptions([...], [...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:07:27 --> Undefined array key "ssl_key"
in SYSTEMPATH\HTTP\CURLRequest.php on line 531.
 1 SYSTEMPATH\HTTP\CURLRequest.php(531): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "ssl_key"', 'SYSTEMPATH\\HTTP\\CURLRequest.php', 531)
 2 SYSTEMPATH\HTTP\CURLRequest.php(363): CodeIgniter\HTTP\CURLRequest->setCURLOptions([...], [...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:07:31 --> Undefined array key "ssl_key"
in SYSTEMPATH\HTTP\CURLRequest.php on line 531.
 1 SYSTEMPATH\HTTP\CURLRequest.php(531): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "ssl_key"', 'SYSTEMPATH\\HTTP\\CURLRequest.php', 531)
 2 SYSTEMPATH\HTTP\CURLRequest.php(363): CodeIgniter\HTTP\CURLRequest->setCURLOptions([...], [...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:10:23 --> Undefined constant "App\Controllers\ThirdParty"
in APPPATH\Controllers\BaseController.php on line 79.
 1 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 2 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:11:37 --> Undefined array key "ssl_key"
in SYSTEMPATH\HTTP\CURLRequest.php on line 531.
 1 SYSTEMPATH\HTTP\CURLRequest.php(531): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "ssl_key"', 'SYSTEMPATH\\HTTP\\CURLRequest.php', 531)
 2 SYSTEMPATH\HTTP\CURLRequest.php(363): CodeIgniter\HTTP\CURLRequest->setCURLOptions([...], [...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:11:40 --> Undefined array key "ssl_key"
in SYSTEMPATH\HTTP\CURLRequest.php on line 531.
 1 SYSTEMPATH\HTTP\CURLRequest.php(531): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "ssl_key"', 'SYSTEMPATH\\HTTP\\CURLRequest.php', 531)
 2 SYSTEMPATH\HTTP\CURLRequest.php(363): CodeIgniter\HTTP\CURLRequest->setCURLOptions([...], [...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:11:59 --> syntax error, unexpected variable "$response", expecting "," or ";"
in APPPATH\Controllers\BaseController.php on line 77.
 1 SYSTEMPATH\Autoloader\Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile('APPPATH\\Controllers\\BaseController.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\BaseController')
 3 APPPATH\Controllers\Home.php(6): CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\BaseController')
 4 SYSTEMPATH\Autoloader\Autoloader.php(312): include_once('APPPATH\\Controllers\\Home.php')
 5 SYSTEMPATH\Autoloader\Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile('APPPATH\\Controllers\\Home.php')
 6 SYSTEMPATH\Autoloader\Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Home')
 7 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Home')
 8 SYSTEMPATH\CodeIgniter.php(892): class_exists('\\App\\Controllers\\Home', true)
 9 SYSTEMPATH\CodeIgniter.php(486): CodeIgniter\CodeIgniter->startController()
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:12:04 --> Undefined array key "ssl_key"
in SYSTEMPATH\HTTP\CURLRequest.php on line 531.
 1 SYSTEMPATH\HTTP\CURLRequest.php(531): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "ssl_key"', 'SYSTEMPATH\\HTTP\\CURLRequest.php', 531)
 2 SYSTEMPATH\HTTP\CURLRequest.php(363): CodeIgniter\HTTP\CURLRequest->setCURLOptions([...], [...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:12:22 --> Undefined array key "ssl_key"
in SYSTEMPATH\HTTP\CURLRequest.php on line 531.
 1 SYSTEMPATH\HTTP\CURLRequest.php(531): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "ssl_key"', 'SYSTEMPATH\\HTTP\\CURLRequest.php', 531)
 2 SYSTEMPATH\HTTP\CURLRequest.php(363): CodeIgniter\HTTP\CURLRequest->setCURLOptions([...], [...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:12:43 --> Undefined array key "ssl_key"
in SYSTEMPATH\HTTP\CURLRequest.php on line 531.
 1 SYSTEMPATH\HTTP\CURLRequest.php(531): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "ssl_key"', 'SYSTEMPATH\\HTTP\\CURLRequest.php', 531)
 2 SYSTEMPATH\HTTP\CURLRequest.php(363): CodeIgniter\HTTP\CURLRequest->setCURLOptions([...], [...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:13:23 --> Undefined array key "ssl_key"
in SYSTEMPATH\HTTP\CURLRequest.php on line 531.
 1 SYSTEMPATH\HTTP\CURLRequest.php(531): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "ssl_key"', 'SYSTEMPATH\\HTTP\\CURLRequest.php', 531)
 2 SYSTEMPATH\HTTP\CURLRequest.php(363): CodeIgniter\HTTP\CURLRequest->setCURLOptions([...], [...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:17:42 --> 22 : The requested URL returned error: 403
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 403')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:17:58 --> 22 : The requested URL returned error: 403
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 403')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:23:46 --> 22 : The requested URL returned error: 403
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 403')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:23:55 --> Undefined array key "ssl_key"
in SYSTEMPATH\HTTP\CURLRequest.php on line 531.
 1 SYSTEMPATH\HTTP\CURLRequest.php(531): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "ssl_key"', 'SYSTEMPATH\\HTTP\\CURLRequest.php', 531)
 2 SYSTEMPATH\HTTP\CURLRequest.php(363): CodeIgniter\HTTP\CURLRequest->setCURLOptions([...], [...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:24:44 --> Undefined array key "ssl_key"
in SYSTEMPATH\HTTP\CURLRequest.php on line 531.
 1 SYSTEMPATH\HTTP\CURLRequest.php(531): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "ssl_key"', 'SYSTEMPATH\\HTTP\\CURLRequest.php', 531)
 2 SYSTEMPATH\HTTP\CURLRequest.php(363): CodeIgniter\HTTP\CURLRequest->setCURLOptions([...], [...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:25:00 --> 60 : SSL certificate problem: unable to get local issuer certificate
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('60', 'SSL certificate problem: unable to get local issuer certificate')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:26:06 --> Undefined array key "ssl_key"
in SYSTEMPATH\HTTP\CURLRequest.php on line 531.
 1 SYSTEMPATH\HTTP\CURLRequest.php(531): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "ssl_key"', 'SYSTEMPATH\\HTTP\\CURLRequest.php', 531)
 2 SYSTEMPATH\HTTP\CURLRequest.php(363): CodeIgniter\HTTP\CURLRequest->setCURLOptions([...], [...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:33:09 --> 60 : SSL certificate problem: unable to get local issuer certificate
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('60', 'SSL certificate problem: unable to get local issuer certificate')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:33:30 --> 60 : SSL certificate problem: unable to get local issuer certificate
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('60', 'SSL certificate problem: unable to get local issuer certificate')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:35:31 --> 60 : SSL certificate problem: unable to get local issuer certificate
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('60', 'SSL certificate problem: unable to get local issuer certificate')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:35:51 --> 22 : The requested URL returned error: 403
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 403')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:35:58 --> 22 : The requested URL returned error: 403
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 403')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:36:55 --> 22 : The requested URL returned error: 403
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 403')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:37:06 --> 22 : The requested URL returned error: 403
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 403')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('GET', 'https://api.github.com/user')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('GET', 'https://api.github.com/user', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:48:27 --> 22 : The requested URL returned error: 415
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 415')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('POST', 'https://postulaciones.solutoria.cl/api/acceso')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('POST', 'https://postulaciones.solutoria.cl/api/acceso', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:50:33 --> 22 : The requested URL returned error: 415
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 415')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('POST', 'https://postulaciones.solutoria.cl/api/acceso')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('POST', 'https://postulaciones.solutoria.cl/api/acceso', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:50:40 --> 22 : The requested URL returned error: 415
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 415')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('POST', 'https://postulaciones.solutoria.cl/api/acceso')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('POST', 'https://postulaciones.solutoria.cl/api/acceso', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:52:21 --> 22 : The requested URL returned error: 415
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 415')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('POST', 'https://postulaciones.solutoria.cl/api/acceso')
 4 APPPATH\Controllers\BaseController.php(77): CodeIgniter\HTTP\CURLRequest->request('POST', 'https://postulaciones.solutoria.cl/api/acceso', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:57:48 --> Call to a member function getStatusCode() on string
in APPPATH\Controllers\BaseController.php on line 101.
 1 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 2 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 19:58:08 --> Call to a member function getBody() on string
in APPPATH\Controllers\BaseController.php on line 102.
 1 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 2 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 20:22:32 --> 22 : The requested URL returned error: 401
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 401')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('get', 'https://postulaciones.solutoria.cl/api/indicadores')
 4 SYSTEMPATH\HTTP\CURLRequest.php(167): CodeIgniter\HTTP\CURLRequest->request('get', 'https://postulaciones.solutoria.cl/api/indicadores', [...])
 5 APPPATH\Controllers\BaseController.php(83): CodeIgniter\HTTP\CURLRequest->get('https://postulaciones.solutoria.cl/api/indicadores', [...])
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 20:22:56 --> 22 : The requested URL returned error: 401
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 401')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('get', 'https://postulaciones.solutoria.cl/api/indicadores')
 4 SYSTEMPATH\HTTP\CURLRequest.php(167): CodeIgniter\HTTP\CURLRequest->request('get', 'https://postulaciones.solutoria.cl/api/indicadores', [...])
 5 APPPATH\Controllers\BaseController.php(83): CodeIgniter\HTTP\CURLRequest->get('https://postulaciones.solutoria.cl/api/indicadores', [...])
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 20:27:19 --> 22 : The requested URL returned error: 401
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 401')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('get', 'https://postulaciones.solutoria.cl/api/indicadores')
 4 SYSTEMPATH\HTTP\CURLRequest.php(167): CodeIgniter\HTTP\CURLRequest->request('get', 'https://postulaciones.solutoria.cl/api/indicadores', [...])
 5 APPPATH\Controllers\BaseController.php(83): CodeIgniter\HTTP\CURLRequest->get('https://postulaciones.solutoria.cl/api/indicadores', [...])
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 20:28:01 --> 22 : The requested URL returned error: 401
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 401')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('get', 'https://postulaciones.solutoria.cl/api/indicadores')
 4 SYSTEMPATH\HTTP\CURLRequest.php(167): CodeIgniter\HTTP\CURLRequest->request('get', 'https://postulaciones.solutoria.cl/api/indicadores', [...])
 5 APPPATH\Controllers\BaseController.php(83): CodeIgniter\HTTP\CURLRequest->get('https://postulaciones.solutoria.cl/api/indicadores', [...])
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 20:28:34 --> 22 : The requested URL returned error: 401
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 401')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('get', 'https://postulaciones.solutoria.cl/api/indicadores')
 4 SYSTEMPATH\HTTP\CURLRequest.php(167): CodeIgniter\HTTP\CURLRequest->request('get', 'https://postulaciones.solutoria.cl/api/indicadores', [...])
 5 APPPATH\Controllers\BaseController.php(83): CodeIgniter\HTTP\CURLRequest->get('https://postulaciones.solutoria.cl/api/indicadores', [...])
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 20:28:47 --> 22 : The requested URL returned error: 401
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 401')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('get', 'https://postulaciones.solutoria.cl/api/indicadores')
 4 SYSTEMPATH\HTTP\CURLRequest.php(167): CodeIgniter\HTTP\CURLRequest->request('get', 'https://postulaciones.solutoria.cl/api/indicadores', [...])
 5 APPPATH\Controllers\BaseController.php(83): CodeIgniter\HTTP\CURLRequest->get('https://postulaciones.solutoria.cl/api/indicadores', [...])
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 20:28:52 --> 22 : The requested URL returned error: 401
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 401')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('get', 'https://postulaciones.solutoria.cl/api/indicadores')
 4 SYSTEMPATH\HTTP\CURLRequest.php(167): CodeIgniter\HTTP\CURLRequest->request('get', 'https://postulaciones.solutoria.cl/api/indicadores', [...])
 5 APPPATH\Controllers\BaseController.php(83): CodeIgniter\HTTP\CURLRequest->get('https://postulaciones.solutoria.cl/api/indicadores', [...])
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 20:29:00 --> 22 : The requested URL returned error: 401
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 401')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('get', 'https://postulaciones.solutoria.cl/api/indicadores')
 4 SYSTEMPATH\HTTP\CURLRequest.php(167): CodeIgniter\HTTP\CURLRequest->request('get', 'https://postulaciones.solutoria.cl/api/indicadores', [...])
 5 APPPATH\Controllers\BaseController.php(83): CodeIgniter\HTTP\CURLRequest->get('https://postulaciones.solutoria.cl/api/indicadores', [...])
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 20:29:39 --> 22 : The requested URL returned error: 401
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 401')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('get', 'https://postulaciones.solutoria.cl/api/indicadores')
 4 SYSTEMPATH\HTTP\CURLRequest.php(167): CodeIgniter\HTTP\CURLRequest->request('get', 'https://postulaciones.solutoria.cl/api/indicadores', [...])
 5 APPPATH\Controllers\BaseController.php(83): CodeIgniter\HTTP\CURLRequest->get('https://postulaciones.solutoria.cl/api/indicadores', [...])
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 20:35:14 --> 22 : The requested URL returned error: 401
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 401')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('get', 'https://postulaciones.solutoria.cl/api/indicadores')
 4 SYSTEMPATH\HTTP\CURLRequest.php(167): CodeIgniter\HTTP\CURLRequest->request('get', 'https://postulaciones.solutoria.cl/api/indicadores', [...])
 5 APPPATH\Controllers\BaseController.php(83): CodeIgniter\HTTP\CURLRequest->get('https://postulaciones.solutoria.cl/api/indicadores', [...])
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 20:38:22 --> 22 : The requested URL returned error: 401
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 401')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('get', 'https://postulaciones.solutoria.cl/api/indicadores')
 4 SYSTEMPATH\HTTP\CURLRequest.php(167): CodeIgniter\HTTP\CURLRequest->request('get', 'https://postulaciones.solutoria.cl/api/indicadores', [...])
 5 APPPATH\Controllers\BaseController.php(83): CodeIgniter\HTTP\CURLRequest->get('https://postulaciones.solutoria.cl/api/indicadores', [...])
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 20:38:36 --> 22 : The requested URL returned error: 401
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 401')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('get', 'https://postulaciones.solutoria.cl/api/indicadores')
 4 SYSTEMPATH\HTTP\CURLRequest.php(167): CodeIgniter\HTTP\CURLRequest->request('get', 'https://postulaciones.solutoria.cl/api/indicadores', [...])
 5 APPPATH\Controllers\BaseController.php(83): CodeIgniter\HTTP\CURLRequest->get('https://postulaciones.solutoria.cl/api/indicadores', [...])
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 20:39:06 --> 22 : The requested URL returned error: 401
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 401')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('get', 'https://postulaciones.solutoria.cl/api/indicadores')
 4 SYSTEMPATH\HTTP\CURLRequest.php(167): CodeIgniter\HTTP\CURLRequest->request('get', 'https://postulaciones.solutoria.cl/api/indicadores', [...])
 5 APPPATH\Controllers\BaseController.php(83): CodeIgniter\HTTP\CURLRequest->get('https://postulaciones.solutoria.cl/api/indicadores', [...])
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 20:39:40 --> 22 : The requested URL returned error: 401
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 401')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('get', 'https://postulaciones.solutoria.cl/api/indicadores')
 4 SYSTEMPATH\HTTP\CURLRequest.php(167): CodeIgniter\HTTP\CURLRequest->request('get', 'https://postulaciones.solutoria.cl/api/indicadores', [...])
 5 APPPATH\Controllers\BaseController.php(83): CodeIgniter\HTTP\CURLRequest->get('https://postulaciones.solutoria.cl/api/indicadores', [...])
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 20:40:16 --> 22 : The requested URL returned error: 401
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 401')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('get', 'https://postulaciones.solutoria.cl/api/indicadores')
 4 SYSTEMPATH\HTTP\CURLRequest.php(167): CodeIgniter\HTTP\CURLRequest->request('get', 'https://postulaciones.solutoria.cl/api/indicadores', [...])
 5 APPPATH\Controllers\BaseController.php(83): CodeIgniter\HTTP\CURLRequest->get('https://postulaciones.solutoria.cl/api/indicadores', [...])
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 20:43:17 --> Undefined constant "App\Controllers\get"
in APPPATH\Controllers\BaseController.php on line 83.
 1 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 2 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 20:43:30 --> 22 : The requested URL returned error: 401
in SYSTEMPATH\HTTP\CURLRequest.php on line 666.
 1 SYSTEMPATH\HTTP\CURLRequest.php(666): CodeIgniter\HTTP\Exceptions\HTTPException::forCurlError('22', 'The requested URL returned error: 401')
 2 SYSTEMPATH\HTTP\CURLRequest.php(372): CodeIgniter\HTTP\CURLRequest->sendRequest([...])
 3 SYSTEMPATH\HTTP\CURLRequest.php(134): CodeIgniter\HTTP\CURLRequest->send('get', 'https://postulaciones.solutoria.cl/api/indicadores')
 4 APPPATH\Controllers\BaseController.php(83): CodeIgniter\HTTP\CURLRequest->request('get', 'https://postulaciones.solutoria.cl/api/indicadores', [...])
 5 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 6 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 21:03:24 --> Array to string conversion
in APPPATH\Controllers\BaseController.php on line 91.
 1 APPPATH\Controllers\BaseController.php(91): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'APPPATH\\Controllers\\BaseController.php', 91)
 2 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 3 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 21:25:46 --> Object of class stdClass could not be converted to string
in APPPATH\Controllers\BaseController.php on line 92.
 1 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 2 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 21:26:16 --> Cannot use object of type stdClass as array
in APPPATH\Controllers\BaseController.php on line 92.
 1 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 2 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 21:48:05 --> Undefined array key "nombreindicador"
in APPPATH\Controllers\BaseController.php on line 96.
 1 APPPATH\Controllers\BaseController.php(96): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "nombreindicador"', 'APPPATH\\Controllers\\BaseController.php', 96)
 2 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 3 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 21:51:04 --> Undefined property: App\Controllers\Home::$db
in APPPATH\Controllers\BaseController.php on line 95.
 1 APPPATH\Controllers\BaseController.php(95): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Controllers\\Home::$db', 'APPPATH\\Controllers\\BaseController.php', 95)
 2 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 3 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 21:51:26 --> Undefined array key "codigoindicador"
in APPPATH\Controllers\BaseController.php on line 97.
 1 APPPATH\Controllers\BaseController.php(97): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "codigoindicador"', 'APPPATH\\Controllers\\BaseController.php', 97)
 2 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 3 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 21:54:51 --> Undefined array key "unidadmedidaIndicador"
in APPPATH\Controllers\BaseController.php on line 102.
 1 APPPATH\Controllers\BaseController.php(102): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "unidadmedidaIndicador"', 'APPPATH\\Controllers\\BaseController.php', 102)
 2 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 3 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-02-02 21:55:22 --> ErrorException: pg_query(): Query failed: ERROR:  no existe la relación «TareaAPIDB»
LINE 1: INSERT INTO &quot;TareaAPIDB&quot; (&quot;id&quot;, &quot;nombreindicador&quot;, &quot;codigoin...
                    ^ in E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php:147
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'E:\\Codigo\\Tarea...', 147)
#1 E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php(147): pg_query(Object(PgSql\Connection), 'INSERT INTO "Ta...')
#2 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(691): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "Ta...')
#3 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "Ta...')
#4 E:\Codigo\TareaAPI\system\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "Ta...', Array, false)
#5 E:\Codigo\TareaAPI\app\Controllers\BaseController.php(107): CodeIgniter\Database\BaseBuilder->insert(Array)
#6 E:\Codigo\TareaAPI\system\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#7 E:\Codigo\TareaAPI\system\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
#8 E:\Codigo\TareaAPI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 E:\Codigo\TareaAPI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 E:\Codigo\TareaAPI\system\Commands\Server\rewrite.php(47): require_once('E:\\Codigo\\Tarea...')
#11 {main}
CRITICAL - 2023-02-02 21:55:22 --> pg_query(): Query failed: ERROR:  no existe la relación «TareaAPIDB»
LINE 1: INSERT INTO &quot;TareaAPIDB&quot; (&quot;id&quot;, &quot;nombreindicador&quot;, &quot;codigoin...
                    ^
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "TareaAPIDB" ("id", "nombreindicador", "codigoindicador", "unidadmedidaindicador", "valorindicador", "fechaindicador") VALUES (:id:, :nombreindicador:, :codigoindicador:, :unidadmedidaindicador:, :valorindicador:, :fechaindicador:)', [...], false)
 2 APPPATH\Controllers\BaseController.php(107): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 4 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-02-02 22:10:54 --> ErrorException: pg_query(): Query failed: ERROR:  el valor nulo en la columna «id» de la relación «indicadorfinancieros» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (null, UNIDAD DE FOMENTO (UF), UF, null, 29061.89, 2021-01-09, 2023-02-02 22:10:54, 2023-02-02 22:10:54). in E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php:147
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'E:\\Codigo\\Tarea...', 147)
#1 E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php(147): pg_query(Object(PgSql\Connection), 'INSERT INTO "in...')
#2 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(691): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "in...')
#3 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "in...')
#4 E:\Codigo\TareaAPI\system\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "in...', Array, false)
#5 E:\Codigo\TareaAPI\system\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
#6 E:\Codigo\TareaAPI\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#7 E:\Codigo\TareaAPI\system\Model.php(729): CodeIgniter\BaseModel->insert(Array, false)
#8 E:\Codigo\TareaAPI\app\Controllers\BaseController.php(110): CodeIgniter\Model->insert(Array, false)
#9 E:\Codigo\TareaAPI\system\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#10 E:\Codigo\TareaAPI\system\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
#11 E:\Codigo\TareaAPI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\Codigo\TareaAPI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 E:\Codigo\TareaAPI\system\Commands\Server\rewrite.php(47): require_once('E:\\Codigo\\Tarea...')
#14 {main}
CRITICAL - 2023-02-02 22:10:54 --> pg_query(): Query failed: ERROR:  el valor nulo en la columna «id» de la relación «indicadorfinancieros» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (null, UNIDAD DE FOMENTO (UF), UF, null, 29061.89, 2021-01-09, 2023-02-02 22:10:54, 2023-02-02 22:10:54).
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "indicadorfinancieros" ("nombreindicador", "codigoindicador", "valorindicador", "fechaindicador", "created_at", "updated_at") VALUES (:nombreindicador:, :codigoindicador:, :valorindicador:, :fechaindicador:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(729): CodeIgniter\BaseModel->insert([...], false)
 5 APPPATH\Controllers\BaseController.php(110): CodeIgniter\Model->insert([...], false)
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 22:11:44 --> Undefined array key "Id"
in APPPATH\Controllers\BaseController.php on line 102.
 1 APPPATH\Controllers\BaseController.php(102): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "Id"', 'APPPATH\\Controllers\\BaseController.php', 102)
 2 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 3 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-02-02 22:11:57 --> ErrorException: pg_query(): Query failed: ERROR:  el valor nulo en la columna «id» de la relación «indicadorfinancieros» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (null, UNIDAD DE FOMENTO (UF), UF, null, 29061.89, 2021-01-09, 2023-02-02 22:11:57, 2023-02-02 22:11:57). in E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php:147
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'E:\\Codigo\\Tarea...', 147)
#1 E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php(147): pg_query(Object(PgSql\Connection), 'INSERT INTO "in...')
#2 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(691): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "in...')
#3 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "in...')
#4 E:\Codigo\TareaAPI\system\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "in...', Array, false)
#5 E:\Codigo\TareaAPI\system\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
#6 E:\Codigo\TareaAPI\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#7 E:\Codigo\TareaAPI\system\Model.php(729): CodeIgniter\BaseModel->insert(Array, false)
#8 E:\Codigo\TareaAPI\app\Controllers\BaseController.php(110): CodeIgniter\Model->insert(Array, false)
#9 E:\Codigo\TareaAPI\system\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#10 E:\Codigo\TareaAPI\system\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
#11 E:\Codigo\TareaAPI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\Codigo\TareaAPI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 E:\Codigo\TareaAPI\system\Commands\Server\rewrite.php(47): require_once('E:\\Codigo\\Tarea...')
#14 {main}
CRITICAL - 2023-02-02 22:11:57 --> pg_query(): Query failed: ERROR:  el valor nulo en la columna «id» de la relación «indicadorfinancieros» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (null, UNIDAD DE FOMENTO (UF), UF, null, 29061.89, 2021-01-09, 2023-02-02 22:11:57, 2023-02-02 22:11:57).
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "indicadorfinancieros" ("nombreindicador", "codigoindicador", "valorindicador", "fechaindicador", "created_at", "updated_at") VALUES (:nombreindicador:, :codigoindicador:, :valorindicador:, :fechaindicador:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(729): CodeIgniter\BaseModel->insert([...], false)
 5 APPPATH\Controllers\BaseController.php(110): CodeIgniter\Model->insert([...], false)
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 22:12:46 --> Array to string conversion
in APPPATH\Controllers\BaseController.php on line 100.
 1 APPPATH\Controllers\BaseController.php(100): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'APPPATH\\Controllers\\BaseController.php', 100)
 2 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 3 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-02-02 22:13:17 --> ErrorException: pg_query(): Query failed: ERROR:  el valor nulo en la columna «id» de la relación «indicadorfinancieros» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (null, UNIDAD DE FOMENTO (UF), UF, null, 29061.89, 2021-01-09, 2023-02-02 22:13:17, 2023-02-02 22:13:17). in E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php:147
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'E:\\Codigo\\Tarea...', 147)
#1 E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php(147): pg_query(Object(PgSql\Connection), 'INSERT INTO "in...')
#2 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(691): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "in...')
#3 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "in...')
#4 E:\Codigo\TareaAPI\system\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "in...', Array, false)
#5 E:\Codigo\TareaAPI\system\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
#6 E:\Codigo\TareaAPI\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#7 E:\Codigo\TareaAPI\system\Model.php(729): CodeIgniter\BaseModel->insert(Array, false)
#8 E:\Codigo\TareaAPI\app\Controllers\BaseController.php(110): CodeIgniter\Model->insert(Array, false)
#9 E:\Codigo\TareaAPI\system\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#10 E:\Codigo\TareaAPI\system\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
#11 E:\Codigo\TareaAPI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\Codigo\TareaAPI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 E:\Codigo\TareaAPI\system\Commands\Server\rewrite.php(47): require_once('E:\\Codigo\\Tarea...')
#14 {main}
CRITICAL - 2023-02-02 22:13:17 --> pg_query(): Query failed: ERROR:  el valor nulo en la columna «id» de la relación «indicadorfinancieros» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (null, UNIDAD DE FOMENTO (UF), UF, null, 29061.89, 2021-01-09, 2023-02-02 22:13:17, 2023-02-02 22:13:17).
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "indicadorfinancieros" ("nombreindicador", "codigoindicador", "valorindicador", "fechaindicador", "created_at", "updated_at") VALUES (:nombreindicador:, :codigoindicador:, :valorindicador:, :fechaindicador:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(729): CodeIgniter\BaseModel->insert([...], false)
 5 APPPATH\Controllers\BaseController.php(110): CodeIgniter\Model->insert([...], false)
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-02-02 22:15:38 --> ErrorException: pg_query(): Query failed: ERROR:  el valor nulo en la columna «id» de la relación «indicadorfinancieros» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (null, UNIDAD DE FOMENTO (UF), UF, null, 29061.89, 2021-01-09, 2023-02-02 22:15:38, 2023-02-02 22:15:38). in E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php:147
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'E:\\Codigo\\Tarea...', 147)
#1 E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php(147): pg_query(Object(PgSql\Connection), 'INSERT INTO "in...')
#2 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(691): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "in...')
#3 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "in...')
#4 E:\Codigo\TareaAPI\system\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "in...', Array, false)
#5 E:\Codigo\TareaAPI\system\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
#6 E:\Codigo\TareaAPI\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#7 E:\Codigo\TareaAPI\system\Model.php(729): CodeIgniter\BaseModel->insert(Array, false)
#8 E:\Codigo\TareaAPI\app\Controllers\BaseController.php(110): CodeIgniter\Model->insert(Array, false)
#9 E:\Codigo\TareaAPI\system\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#10 E:\Codigo\TareaAPI\system\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
#11 E:\Codigo\TareaAPI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\Codigo\TareaAPI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 E:\Codigo\TareaAPI\system\Commands\Server\rewrite.php(47): require_once('E:\\Codigo\\Tarea...')
#14 {main}
CRITICAL - 2023-02-02 22:15:38 --> pg_query(): Query failed: ERROR:  el valor nulo en la columna «id» de la relación «indicadorfinancieros» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (null, UNIDAD DE FOMENTO (UF), UF, null, 29061.89, 2021-01-09, 2023-02-02 22:15:38, 2023-02-02 22:15:38).
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "indicadorfinancieros" ("nombreindicador", "codigoindicador", "valorindicador", "fechaindicador", "created_at", "updated_at") VALUES (:nombreindicador:, :codigoindicador:, :valorindicador:, :fechaindicador:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(729): CodeIgniter\BaseModel->insert([...], false)
 5 APPPATH\Controllers\BaseController.php(110): CodeIgniter\Model->insert([...], false)
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-02-02 22:17:23 --> ErrorException: pg_query(): Query failed: ERROR:  el valor nulo en la columna «id» de la relación «indicadorfinancieros» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (null, UNIDAD DE FOMENTO (UF), UF, null, 29061.89, 2021-01-09, 2023-02-02 22:17:23, 2023-02-02 22:17:23). in E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php:147
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'E:\\Codigo\\Tarea...', 147)
#1 E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php(147): pg_query(Object(PgSql\Connection), 'INSERT INTO "in...')
#2 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(691): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "in...')
#3 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "in...')
#4 E:\Codigo\TareaAPI\system\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "in...', Array, false)
#5 E:\Codigo\TareaAPI\system\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
#6 E:\Codigo\TareaAPI\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#7 E:\Codigo\TareaAPI\system\Model.php(729): CodeIgniter\BaseModel->insert(Array, false)
#8 E:\Codigo\TareaAPI\app\Controllers\BaseController.php(115): CodeIgniter\Model->insert(Array, false)
#9 E:\Codigo\TareaAPI\system\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#10 E:\Codigo\TareaAPI\system\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
#11 E:\Codigo\TareaAPI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\Codigo\TareaAPI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 E:\Codigo\TareaAPI\system\Commands\Server\rewrite.php(47): require_once('E:\\Codigo\\Tarea...')
#14 {main}
CRITICAL - 2023-02-02 22:17:23 --> pg_query(): Query failed: ERROR:  el valor nulo en la columna «id» de la relación «indicadorfinancieros» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (null, UNIDAD DE FOMENTO (UF), UF, null, 29061.89, 2021-01-09, 2023-02-02 22:17:23, 2023-02-02 22:17:23).
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "indicadorfinancieros" ("nombreindicador", "codigoindicador", "valorindicador", "fechaindicador", "created_at", "updated_at") VALUES (:nombreindicador:, :codigoindicador:, :valorindicador:, :fechaindicador:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(729): CodeIgniter\BaseModel->insert([...], false)
 5 APPPATH\Controllers\BaseController.php(115): CodeIgniter\Model->insert([...], false)
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-02-02 22:18:59 --> ErrorException: pg_query(): Query failed: ERROR:  el valor nulo en la columna «id» de la relación «indicadorfinancieros» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (null, UNIDAD DE FOMENTO (UF), UF, null, 29061.89, 2021-01-09, 2023-02-02 22:18:59, 2023-02-02 22:18:59). in E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php:147
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'E:\\Codigo\\Tarea...', 147)
#1 E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php(147): pg_query(Object(PgSql\Connection), 'INSERT INTO "in...')
#2 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(691): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "in...')
#3 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "in...')
#4 E:\Codigo\TareaAPI\system\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "in...', Array, false)
#5 E:\Codigo\TareaAPI\system\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
#6 E:\Codigo\TareaAPI\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#7 E:\Codigo\TareaAPI\system\Model.php(729): CodeIgniter\BaseModel->insert(Array, false)
#8 E:\Codigo\TareaAPI\app\Controllers\BaseController.php(115): CodeIgniter\Model->insert(Array, false)
#9 E:\Codigo\TareaAPI\system\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#10 E:\Codigo\TareaAPI\system\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
#11 E:\Codigo\TareaAPI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\Codigo\TareaAPI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 E:\Codigo\TareaAPI\system\Commands\Server\rewrite.php(47): require_once('E:\\Codigo\\Tarea...')
#14 {main}
CRITICAL - 2023-02-02 22:18:59 --> pg_query(): Query failed: ERROR:  el valor nulo en la columna «id» de la relación «indicadorfinancieros» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (null, UNIDAD DE FOMENTO (UF), UF, null, 29061.89, 2021-01-09, 2023-02-02 22:18:59, 2023-02-02 22:18:59).
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "indicadorfinancieros" ("nombreindicador", "codigoindicador", "valorindicador", "fechaindicador", "created_at", "updated_at") VALUES (:nombreindicador:, :codigoindicador:, :valorindicador:, :fechaindicador:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(729): CodeIgniter\BaseModel->insert([...], false)
 5 APPPATH\Controllers\BaseController.php(115): CodeIgniter\Model->insert([...], false)
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-02-02 22:33:08 --> ErrorException: pg_query(): Query failed: ERROR:  el valor nulo en la columna «unidadmedidaindicador» de la relación «indicadorfinancieros» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (6, UNIDAD DE FOMENTO (UF), UF, null, 29061.89, 2021-01-09, 2023-02-02 22:33:08, 2023-02-02 22:33:08). in E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php:147
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'E:\\Codigo\\Tarea...', 147)
#1 E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php(147): pg_query(Object(PgSql\Connection), 'INSERT INTO "in...')
#2 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(691): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "in...')
#3 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "in...')
#4 E:\Codigo\TareaAPI\system\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "in...', Array, false)
#5 E:\Codigo\TareaAPI\system\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
#6 E:\Codigo\TareaAPI\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#7 E:\Codigo\TareaAPI\system\Model.php(729): CodeIgniter\BaseModel->insert(Array, true)
#8 E:\Codigo\TareaAPI\app\Controllers\BaseController.php(117): CodeIgniter\Model->insert(Array)
#9 E:\Codigo\TareaAPI\system\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#10 E:\Codigo\TareaAPI\system\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
#11 E:\Codigo\TareaAPI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\Codigo\TareaAPI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 E:\Codigo\TareaAPI\system\Commands\Server\rewrite.php(47): require_once('E:\\Codigo\\Tarea...')
#14 {main}
CRITICAL - 2023-02-02 22:33:08 --> pg_query(): Query failed: ERROR:  el valor nulo en la columna «unidadmedidaindicador» de la relación «indicadorfinancieros» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (6, UNIDAD DE FOMENTO (UF), UF, null, 29061.89, 2021-01-09, 2023-02-02 22:33:08, 2023-02-02 22:33:08).
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "indicadorfinancieros" ("nombreindicador", "codigoindicador", "valorindicador", "fechaindicador", "created_at", "updated_at", "id") VALUES (:nombreindicador:, :codigoindicador:, :valorindicador:, :fechaindicador:, :created_at:, :updated_at:, :id:)', [...], false)
 2 SYSTEMPATH\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(729): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\BaseController.php(117): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-02-02 22:34:58 --> ErrorException: pg_query(): Query failed: ERROR:  el valor nulo en la columna «unidadmedidaindicador» de la relación «indicadorfinancieros» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (6, UNIDAD DE FOMENTO (UF), UF, null, 29061.89, 2021-01-09, 2023-02-02 22:34:58, 2023-02-02 22:34:58). in E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php:147
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'E:\\Codigo\\Tarea...', 147)
#1 E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php(147): pg_query(Object(PgSql\Connection), 'INSERT INTO "in...')
#2 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(691): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "in...')
#3 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "in...')
#4 E:\Codigo\TareaAPI\system\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "in...', Array, false)
#5 E:\Codigo\TareaAPI\system\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
#6 E:\Codigo\TareaAPI\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#7 E:\Codigo\TareaAPI\system\Model.php(729): CodeIgniter\BaseModel->insert(Array, true)
#8 E:\Codigo\TareaAPI\app\Controllers\BaseController.php(117): CodeIgniter\Model->insert(Array)
#9 E:\Codigo\TareaAPI\system\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#10 E:\Codigo\TareaAPI\system\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
#11 E:\Codigo\TareaAPI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\Codigo\TareaAPI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 E:\Codigo\TareaAPI\system\Commands\Server\rewrite.php(47): require_once('E:\\Codigo\\Tarea...')
#14 {main}
CRITICAL - 2023-02-02 22:34:58 --> pg_query(): Query failed: ERROR:  el valor nulo en la columna «unidadmedidaindicador» de la relación «indicadorfinancieros» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (6, UNIDAD DE FOMENTO (UF), UF, null, 29061.89, 2021-01-09, 2023-02-02 22:34:58, 2023-02-02 22:34:58).
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "indicadorfinancieros" ("nombreindicador", "codigoindicador", "valorindicador", "fechaindicador", "created_at", "updated_at", "id") VALUES (:nombreindicador:, :codigoindicador:, :valorindicador:, :fechaindicador:, :created_at:, :updated_at:, :id:)', [...], false)
 2 SYSTEMPATH\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(729): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\BaseController.php(117): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-02-02 22:38:36 --> ErrorException: pg_query(): Query failed: ERROR:  el valor nulo en la columna «unidadmedidaindicador» de la relación «indicadorfinancieros» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (6, UNIDAD DE FOMENTO (UF), UF, null, 29061.89, 2021-01-09, 2023-02-02 22:38:36, 2023-02-02 22:38:36). in E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php:147
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'E:\\Codigo\\Tarea...', 147)
#1 E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php(147): pg_query(Object(PgSql\Connection), 'INSERT INTO "in...')
#2 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(691): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "in...')
#3 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "in...')
#4 E:\Codigo\TareaAPI\system\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "in...', Array, false)
#5 E:\Codigo\TareaAPI\system\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
#6 E:\Codigo\TareaAPI\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#7 E:\Codigo\TareaAPI\system\Model.php(729): CodeIgniter\BaseModel->insert(Array, true)
#8 E:\Codigo\TareaAPI\app\Controllers\BaseController.php(117): CodeIgniter\Model->insert(Array)
#9 E:\Codigo\TareaAPI\system\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#10 E:\Codigo\TareaAPI\system\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
#11 E:\Codigo\TareaAPI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\Codigo\TareaAPI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 E:\Codigo\TareaAPI\system\Commands\Server\rewrite.php(47): require_once('E:\\Codigo\\Tarea...')
#14 {main}
CRITICAL - 2023-02-02 22:38:36 --> pg_query(): Query failed: ERROR:  el valor nulo en la columna «unidadmedidaindicador» de la relación «indicadorfinancieros» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (6, UNIDAD DE FOMENTO (UF), UF, null, 29061.89, 2021-01-09, 2023-02-02 22:38:36, 2023-02-02 22:38:36).
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "indicadorfinancieros" ("nombreindicador", "codigoindicador", "valorindicador", "fechaindicador", "created_at", "updated_at", "id") VALUES (:nombreindicador:, :codigoindicador:, :valorindicador:, :fechaindicador:, :created_at:, :updated_at:, :id:)', [...], false)
 2 SYSTEMPATH\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(729): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\BaseController.php(117): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-02-02 22:41:50 --> ErrorException: pg_query(): Query failed: ERROR:  el valor nulo en la columna «unidadmedidaindicador» de la relación «indicadorfinancieros» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (6, UNIDAD DE FOMENTO (UF), UF, null, 29061.89, 2021-01-09, 2023-02-02 22:41:50, 2023-02-02 22:41:50). in E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php:147
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'E:\\Codigo\\Tarea...', 147)
#1 E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php(147): pg_query(Object(PgSql\Connection), 'INSERT INTO "in...')
#2 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(691): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "in...')
#3 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "in...')
#4 E:\Codigo\TareaAPI\system\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "in...', Array, false)
#5 E:\Codigo\TareaAPI\system\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
#6 E:\Codigo\TareaAPI\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#7 E:\Codigo\TareaAPI\system\Model.php(729): CodeIgniter\BaseModel->insert(Array, true)
#8 E:\Codigo\TareaAPI\app\Controllers\BaseController.php(117): CodeIgniter\Model->insert(Array)
#9 E:\Codigo\TareaAPI\system\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#10 E:\Codigo\TareaAPI\system\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
#11 E:\Codigo\TareaAPI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\Codigo\TareaAPI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 E:\Codigo\TareaAPI\system\Commands\Server\rewrite.php(47): require_once('E:\\Codigo\\Tarea...')
#14 {main}
CRITICAL - 2023-02-02 22:41:50 --> pg_query(): Query failed: ERROR:  el valor nulo en la columna «unidadmedidaindicador» de la relación «indicadorfinancieros» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (6, UNIDAD DE FOMENTO (UF), UF, null, 29061.89, 2021-01-09, 2023-02-02 22:41:50, 2023-02-02 22:41:50).
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "indicadorfinancieros" ("nombreindicador", "codigoindicador", "valorindicador", "fechaindicador", "created_at", "updated_at", "id") VALUES (:nombreindicador:, :codigoindicador:, :valorindicador:, :fechaindicador:, :created_at:, :updated_at:, :id:)', [...], false)
 2 SYSTEMPATH\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(729): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\BaseController.php(117): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-02-02 22:43:09 --> ErrorException: pg_query(): Query failed: ERROR:  el valor nulo en la columna «unidadmedidaindicador» de la relación «indicadorfinancieros» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (6, UNIDAD DE FOMENTO (UF), UF, null, 29061.89, 2021-01-09, 2023-02-02 22:43:09, 2023-02-02 22:43:09). in E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php:147
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'E:\\Codigo\\Tarea...', 147)
#1 E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php(147): pg_query(Object(PgSql\Connection), 'INSERT INTO "in...')
#2 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(691): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "in...')
#3 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "in...')
#4 E:\Codigo\TareaAPI\system\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "in...', Array, false)
#5 E:\Codigo\TareaAPI\system\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
#6 E:\Codigo\TareaAPI\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#7 E:\Codigo\TareaAPI\system\Model.php(729): CodeIgniter\BaseModel->insert(Array, true)
#8 E:\Codigo\TareaAPI\app\Controllers\BaseController.php(117): CodeIgniter\Model->insert(Array)
#9 E:\Codigo\TareaAPI\system\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#10 E:\Codigo\TareaAPI\system\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
#11 E:\Codigo\TareaAPI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\Codigo\TareaAPI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 E:\Codigo\TareaAPI\system\Commands\Server\rewrite.php(47): require_once('E:\\Codigo\\Tarea...')
#14 {main}
CRITICAL - 2023-02-02 22:43:09 --> pg_query(): Query failed: ERROR:  el valor nulo en la columna «unidadmedidaindicador» de la relación «indicadorfinancieros» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (6, UNIDAD DE FOMENTO (UF), UF, null, 29061.89, 2021-01-09, 2023-02-02 22:43:09, 2023-02-02 22:43:09).
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "indicadorfinancieros" ("nombreindicador", "codigoindicador", "valorindicador", "fechaindicador", "created_at", "updated_at", "id") VALUES (:nombreindicador:, :codigoindicador:, :valorindicador:, :fechaindicador:, :created_at:, :updated_at:, :id:)', [...], false)
 2 SYSTEMPATH\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(729): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\BaseController.php(117): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-02-02 22:43:53 --> ErrorException: pg_query(): Query failed: ERROR:  el valor nulo en la columna «unidadmedidaindicador» de la relación «indicadorfinancieros» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (6, UNIDAD DE FOMENTO (UF), UF, null, 29061.89, 2021-01-09, 2023-02-02 22:43:53, 2023-02-02 22:43:53). in E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php:147
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'E:\\Codigo\\Tarea...', 147)
#1 E:\Codigo\TareaAPI\system\Database\Postgre\Connection.php(147): pg_query(Object(PgSql\Connection), 'INSERT INTO "in...')
#2 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(691): CodeIgniter\Database\Postgre\Connection->execute('INSERT INTO "in...')
#3 E:\Codigo\TareaAPI\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO "in...')
#4 E:\Codigo\TareaAPI\system\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "in...', Array, false)
#5 E:\Codigo\TareaAPI\system\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
#6 E:\Codigo\TareaAPI\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#7 E:\Codigo\TareaAPI\system\Model.php(729): CodeIgniter\BaseModel->insert(Array, true)
#8 E:\Codigo\TareaAPI\app\Controllers\BaseController.php(117): CodeIgniter\Model->insert(Array)
#9 E:\Codigo\TareaAPI\system\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#10 E:\Codigo\TareaAPI\system\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
#11 E:\Codigo\TareaAPI\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\Codigo\TareaAPI\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 E:\Codigo\TareaAPI\system\Commands\Server\rewrite.php(47): require_once('E:\\Codigo\\Tarea...')
#14 {main}
CRITICAL - 2023-02-02 22:43:53 --> pg_query(): Query failed: ERROR:  el valor nulo en la columna «unidadmedidaindicador» de la relación «indicadorfinancieros» viola la restricción de no nulo
DETAIL:  La fila que falla contiene (6, UNIDAD DE FOMENTO (UF), UF, null, 29061.89, 2021-01-09, 2023-02-02 22:43:53, 2023-02-02 22:43:53).
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO "indicadorfinancieros" ("nombreindicador", "codigoindicador", "valorindicador", "fechaindicador", "created_at", "updated_at", "id") VALUES (:nombreindicador:, :codigoindicador:, :valorindicador:, :fechaindicador:, :created_at:, :updated_at:, :id:)', [...], false)
 2 SYSTEMPATH\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(729): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\BaseController.php(117): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 23:35:47 --> include(APPPATH/Views/css/header.css): Failed to open stream: No such file or directory
in APPPATH\Views\header.php on line 9.
 1 APPPATH\Views\header.php(9): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(APPPATH/Views/css/header.css): Failed to open stream: No such file or directory', 'APPPATH\\Views\\header.php', 9)
 2 APPPATH\Views\header.php(9): include()
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\header.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('header', [], true)
 6 APPPATH\Controllers\Home.php(10): view('header')
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 23:36:15 --> include(APPPATH\Views\css\header.css): Failed to open stream: No such file or directory
in APPPATH\Views\header.php on line 9.
 1 APPPATH\Views\header.php(9): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(APPPATH\\Views\\css\\header.css): Failed to open stream: No such file or directory', 'APPPATH\\Views\\header.php', 9)
 2 APPPATH\Views\header.php(9): include()
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\header.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('header', [], true)
 6 APPPATH\Controllers\Home.php(10): view('header')
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 23:36:27 --> include(.\css\header.css): Failed to open stream: No such file or directory
in APPPATH\Views\header.php on line 9.
 1 APPPATH\Views\header.php(9): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(.\\css\\header.css): Failed to open stream: No such file or directory', 'APPPATH\\Views\\header.php', 9)
 2 APPPATH\Views\header.php(9): include()
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\header.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('header', [], true)
 6 APPPATH\Controllers\Home.php(10): view('header')
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 23:36:56 --> include(./css/header.css): Failed to open stream: No such file or directory
in APPPATH\Views\header.php on line 9.
 1 APPPATH\Views\header.php(9): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(./css/header.css): Failed to open stream: No such file or directory', 'APPPATH\\Views\\header.php', 9)
 2 APPPATH\Views\header.php(9): include()
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\header.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('header', [], true)
 6 APPPATH\Controllers\Home.php(10): view('header')
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 23:38:22 --> include(./css/header.css): Failed to open stream: No such file or directory
in APPPATH\Views\header.php on line 9.
 1 APPPATH\Views\header.php(9): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(./css/header.css): Failed to open stream: No such file or directory', 'APPPATH\\Views\\header.php', 9)
 2 APPPATH\Views\header.php(9): include()
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\header.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('header', [], true)
 6 APPPATH\Controllers\Home.php(10): view('header')
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 23:38:37 --> include(./css/header.css): Failed to open stream: No such file or directory
in APPPATH\Views\header.php on line 9.
 1 APPPATH\Views\header.php(9): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(./css/header.css): Failed to open stream: No such file or directory', 'APPPATH\\Views\\header.php', 9)
 2 APPPATH\Views\header.php(9): include()
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\header.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('header', [], true)
 6 APPPATH\Controllers\Home.php(10): view('header')
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 23:38:57 --> include(.\css\header.css): Failed to open stream: No such file or directory
in APPPATH\Views\header.php on line 9.
 1 APPPATH\Views\header.php(9): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(.\\css\\header.css): Failed to open stream: No such file or directory', 'APPPATH\\Views\\header.php', 9)
 2 APPPATH\Views\header.php(9): include()
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\header.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('header', [], true)
 6 APPPATH\Controllers\Home.php(10): view('header')
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 23:39:31 --> include(css/main.css): Failed to open stream: No such file or directory
in APPPATH\Views\header.php on line 11.
 1 APPPATH\Views\header.php(11): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(css/main.css): Failed to open stream: No such file or directory', 'APPPATH\\Views\\header.php', 11)
 2 APPPATH\Views\header.php(11): include()
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\header.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('header', [], true)
 6 APPPATH\Controllers\Home.php(10): view('header')
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 23:39:44 --> include(./css/main.css): Failed to open stream: No such file or directory
in APPPATH\Views\header.php on line 11.
 1 APPPATH\Views\header.php(11): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(./css/main.css): Failed to open stream: No such file or directory', 'APPPATH\\Views\\header.php', 11)
 2 APPPATH\Views\header.php(11): include()
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\header.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('header', [], true)
 6 APPPATH\Controllers\Home.php(10): view('header')
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 23:40:01 --> include(./css/header.css): Failed to open stream: No such file or directory
in APPPATH\Views\header.php on line 11.
 1 APPPATH\Views\header.php(11): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(./css/header.css): Failed to open stream: No such file or directory', 'APPPATH\\Views\\header.php', 11)
 2 APPPATH\Views\header.php(11): include()
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\header.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('header', [], true)
 6 APPPATH\Controllers\Home.php(10): view('header')
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-02-02 23:40:09 --> include(./css/header.css): Failed to open stream: No such file or directory
in APPPATH\Views\header.php on line 11.
 1 APPPATH\Views\header.php(11): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(./css/header.css): Failed to open stream: No such file or directory', 'APPPATH\\Views\\header.php', 11)
 2 APPPATH\Views\header.php(11): include()
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\header.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('header', [], true)
 6 APPPATH\Controllers\Home.php(10): view('header')
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
